configurator/src/main.rs

```diff
@@ -163,6 +177,38 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let cli = Cli::parse();
 
     match cli.command {
+        Commands::Difference {
+            pictures,
+            output_dir,
+        } => {
+            let before = opencv::imgcodecs::imread(
+                pictures[0].to_str().expect("Path wasn't valid unicode"),
+                1,
+            )?;
+            let before = cv::Picture::from(before);
+            let after = opencv::imgcodecs::imread(
+                pictures[1].to_str().expect("Path wasn't valid unicode"),
+                1,
+            )?;
+            let after = cv::Picture::from(after);
+
+            if !output_dir.exists() {
+                std::fs::create_dir_all(&output_dir)?;
+            }
+            if !output_dir.is_dir() {
+                Err(Box::new(ConfiguratorError {
```

Comment:
Is the `Box::new` necessary here with the `?`?

Reply:
You're right, it isn't. Removed